apps/site/layouts/layouts.module.css

```
@@ -42,7 +42,11 @@
         p-4
         motion-safe:scroll-smooth
         sm:bg-fixed
-        sm:p-12;
+        sm:p-12
+        lg:sticky
+        lg:bottom-[4rem]
```

```suggestion
        lg:bottom-16
```
In the [default Tailwind spacing](https://github.com/tailwindlabs/tailwindcss/blob/main/stubs/config.full.js#L861) scale, `16` is equivalent to `4rem`